test/reduce_test.cpp

```diff
@@ -353,10 +372,10 @@ struct verify_reduce_with_indices
                                 indices_sizeInBytes,
                                 workspace_dev.get(),
                                 ws_sizeInBytes,
-                                static_cast<const void*>(&alpha),
+                                const_cast<const void*>(alphaPara),
```

Comment:
+1 to @junliume , pls see suggestion below.

Reply:
I guess the compiler always wants us to use  `const void *ptr` in way of `const void *ptr = <xxx>` rather than doing cast at locations where it is used.